apps/finance/app/src/components/Transfers.js

```diff
@@ -162,7 +157,7 @@ const Transfers = React.memo(({ dao, tokens, transactions }) => {
       resolveAddress
     )
     const filename = getDownloadFilename(dao, selectedDateRange)
-    download(data, filename, 'data:text/csv')
+    saveAs(new Blob([data.join('\n')]), filename)
```

Comment:
Should we provide the MIME type?

```suggestion
    saveAs(new Blob([data.join('\n')], {type: "text/csv;charset=utf-8"}), filename)
```